addons/payment_nuvei/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import controllers
+from . import models
+
+from odoo.addons.payment import setup_provider, reset_payment_provider
```

Comment:
Nitpicking:
```suggestion
from odoo.addons.payment import reset_payment_provider, setup_provider
```
Because: `ord('r') < ord('s')`